src/main/java/uk/gov/justice/probation/courtcaseservice/jpa/entity/HearingEntity.java

```diff
@@ -19,21 +19,9 @@
 import uk.gov.justice.probation.courtcaseservice.controller.model.HearingOutcomeItemState;
 import uk.gov.justice.probation.courtcaseservice.service.HearingOutcomeType;
 
-import javax.persistence.CascadeType;
-import javax.persistence.Column;
-import javax.persistence.Entity;
-import javax.persistence.EnumType;
-import javax.persistence.Enumerated;
-import javax.persistence.GeneratedValue;
-import javax.persistence.GenerationType;
-import javax.persistence.Id;
-import javax.persistence.JoinColumn;
-import javax.persistence.ManyToOne;
-import javax.persistence.OneToMany;
-import javax.persistence.OneToOne;
-import javax.persistence.OrderBy;
-import javax.persistence.Table;
+import javax.persistence.*;
```

Comment:
I would've assumed it's best to use named imports, ```*``` means importing everything?

Reply:
Had to use a native query given the complexity is not supported by ORM framework.

Reply:
Yes, IDE did this thing. will revert